tests/express.es5.tests.js

```diff
@@ -161,7 +169,8 @@ describe('expressJwtSecret', () => {
               cb(new Error('This is bad'));
             }
           }
-        })
+        }),
+        algorithms: [ 'RS256' ]
```

Comment:
are these changes to add the `algorithms` array required from the express-jwt upgrade? Just want to make sure it's not a breaking change.

Reply:
Yes, @jimmyjames. That's the actual breaking change announced in the last v6 of `express-jwt` (which we want to upgrade to). See the updated [readme](https://github.com/auth0/express-jwt#required-parameters) and the [cve](https://github.com/advisories/GHSA-6g6m-m6h5-w9gf).

@davidpatrick Changes look good 👍